src/Sql/dbo/Tools/Stored Procedures/Send_Update.sql

```diff
@@ -15,7 +15,8 @@
     @Disabled BIT,
     @HideEmail BIT,
     @CipherId UNIQUEIDENTIFIER = NULL,
-    @Emails NVARCHAR(1024) = NULL
+    @Emails NVARCHAR(4000) = NULL,
+    @AuthType TINYINT = 2
```

Comment:
Default value should be `NULL`

```suggestion
    @AuthType TINYINT = NULL
```